carbon-design-system/carbon-for-ibm-dotcom
# fix(content-group-banner-6794): fix before pseudo-element bug on safari

### Related Ticket(s)

GH Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6794
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2130

### Description

- This PR removes the use of `::before` pseudo-element on components/slots that was previously used to create the divided line because Safari can't recognize the pseudo-elements like `::before` and `::after` in the Shadow Dom.
- 
### Testing Instruction

- Visit the `Content group banner` in Safari- `/?path=/story/components-content-group-banner--default`
- Ensure that the divider line between title and CTA is showing up correctly in Safari on larger breakpoints.

### Expected Display

![Screen Shot 2021-08-11 at 3 23 00 PM](https://user-images.githubusercontent.com/1815714/129090447-c5692387-5637-4da8-94fd-37c09ed9e6b5.png)
